cmd/broker/testdata/kyma-components.yaml

```diff
@@ -1,5 +1,7 @@
 ---
 defaultNamespace: kyma-system
+prerequisites:
+  - name: "cluster-essentials"
 components:
   - name: "ory"
```

Comment:
Why do we still use ory here?

Reply:
It doesn't really matter, tests don't provision anything as provisioner and reconciler are mocked. We just check if configuration sent from KEB matches the configuration other components receive. We can change the components to generic names, like `component1`, `component2`, but the required follow up is changing assertions where configurations are asserted.